.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
       - 'master'
   schedule:
     # Use <https://crontab.guru> to conveniently edit cron schedule.
-    - cron: "0 7 * * *" "30 0 * * 1" # At 07:00 everyday
+    - cron: "0 7 * * *" # At 07:00 everyday
```

Comment:
```suggestion
    - cron: "0 7 * * *" # At 07:00 every day.
```

Reply:
According to manpage

> The CRON_TZ variable specifies the time zone specific for the cron table

But it seems like github runners choose GMT time: https://github.com/actions/virtual-environments/blob/f199e4ad671a97e32b05aec02e87b6c6404caa17/images/macos/provision/configuration/configure-machine.sh#L6-L9